src/domain/LottoPrize.js

```diff
@@ -0,0 +1,38 @@
+export class LottoPrize {
+  #winningNumbers;
+  #bonusNumber;
+
+  constructor({winningNumbers, bonusNumber}) {
+    this.#winningNumbers = winningNumbers;
+    this.#bonusNumber = bonusNumber;
+  }
+
+  #getMatchedCount(lottery) {
+    return lottery.filter(number => this.#winningNumbers.includes(number)).length;
+  }
+  #isBonusMatched(lottery) {
+    return lottery.includes(this.#bonusNumber);
+  }
+
+  getLottoPrize(lottery) {
+    const matchedCount = this.#getMatchedCount(lottery);
+    const isBonusMatched = this.#isBonusMatched(lottery);
+
```

Comment:
```suggestion
    const matchedCount = this.#getMatchedCount(lottery);
    const bonusMatched = this.#isBonusMatched(lottery);
```
여기도 is는 제거해도 될 것 같네요!